crates/ruff_cli/src/main.rs

```diff
@@ -205,7 +205,7 @@ fn check(args: CheckArgs, log_level: LogLevel) -> Result<ExitStatus> {
     let printer = Printer::new(format, log_level, autofix, printer_flags);
 
     if cli.watch {
-        if !matches!(autofix, fix::FixMode::None) {
+        if !autofix.is_none() {
```

Comment:
I actually consider this to be less readable. This now looks like `autofix` is an `Option`.

Reply:
I guess `None` is arguably not a great enum variant name anyway?

Reply:
I removed this commit

Reply:
Thanks. I don't think there's any problem with having `None` as an enum variant name.